presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/README.md

```diff
@@ -0,0 +1,148 @@
+# Presto planner and optimizer tests
+
+Testing planner and optimizer functionality generally requires generating a
+plan and then validating it against a set of expectations. The challenge is
+expressing the expectations in a way that's concise and resilient to changes in
```

Comment:
Suggest:
Expectations should be expressed concisely and be resilient to changes...